components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/config/model/graph/js/JsServletResponse.java

```diff
@@ -83,4 +84,8 @@ private HttpServletResponse getResponse() {
         TransientObjectWrapper<HttpServletResponse> transientObjectWrapper = getWrapped();
         return transientObjectWrapper.getWrapped();
     }
+
+    public void addCookie(Cookie cookie){
+        getResponse().addCookie(cookie);
```

Comment:
What would be the behavior if there's already a cookie with the same name?

Reply:
The old value will be replaced with the new one.